src/app/tabs/camera.cpp

```diff
@@ -0,0 +1,83 @@
+#include <app/config.hpp>
+#include <app/tabs/camera.hpp>
+#include <app/window.hpp>
+#include <app/theme.hpp>
+
+CameraTab::CameraTab(QWidget *parent) : QWidget(parent)
+{
+    settings = Config::get_instance()->get_settings();
+
+    QVBoxLayout *layout = new QVBoxLayout(this);
+
+    player.setVideoOutput(&videoWidget);
+    QString camera = settings->value("cameraName").toString();
+    cameraName = new QLabel(camera, this);
+
+    cameraName->setFont(Theme::font_16);
+       status.setFont(Theme::font_14);

```

Comment:
tabs vs spaces?

Reply:
Yeah, I need to set my editor up to use tabs for my projects and spaces for this one. Manually converting isn't as much fun as it sounds.